src/xunit.performance.api/Scenario.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    public sealed class Scenario : IDisposable
+    {
+        public Scenario(ScenarioConfiguration configuration)
```

Comment:
This looks like a new class.  What was used before this (e.g. what does this replace)?

Reply:
Before we did not have neither scenario nor scenario configuration. We were passing different parameters.

---
In reply to: [143035848](https://github.com/Microsoft/xunit-performance/pull/241#discussion_r143035848) [](ancestors = 143035848)